src/asserts.rs

```diff
@@ -0,0 +1,36 @@
+use core::fmt::Debug;
+
+pub const DEFAULT_PANIC_EXIT_CODE: i8 = -1;
+
+pub static mut __PANIC_EXIT_CODE: i8 = DEFAULT_PANIC_EXIT_CODE;
+
+pub fn set_panic_exit_code(code: i8) {
+    unsafe {
+        __PANIC_EXIT_CODE = code;
+    }
+}
+
+#[macro_export]
+macro_rules! set_error_assert {
+    ($code:expr, $($arg:tt)*) => {{
+        $crate::asserts::set_panic_exit_code($code);
+        assert!($($arg)*);
+        $crate::asserts::set_panic_exit_code($crate::asserts::DEFAULT_PANIC_EXIT_CODE);
+    }};
+}
+
+#[macro_export]
+macro_rules! set_error_assert_eq {
+    ($code:expr, $($arg:tt)*) => {{
+        $crate::asserts::set_panic_exit_code($code);
+        assert_eq!($($arg)*);
+        $crate::asserts::set_panic_exit_code($crate::asserts::DEFAULT_PANIC_EXIT_CODE);
+    }};
+}
+
+pub fn set_error_expect<C: Into<i8>, T, E: Debug>(c: C, r: Result<T, E>, msg: &str) -> T {
```

Comment:
It's better to adding following:
1. Option's expect
2. macro assert_ne!
3. Result's expect_err

Reply:
Done